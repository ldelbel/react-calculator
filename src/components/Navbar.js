import { Link, BrowserRouter as Router } from 'react-router-dom';
import styled from 'styled-components';
import logo from '../assets/math.png';

const Navbar = () => {
  const StyledNav = styled.nav`
    display: flex;
    justify-content: space-between;
    padding: 0 50px;
    background: #ee0979;
    background: -webkit-linear-gradient(to right, #ff6a00, #ee0979);
    background: linear-gradient(to right, #ff6a00, #ee0979);
    height: 50px;

    img {
      height: 40px;
      align-self: center;
    }

    a {
      color: white;
      text-decoration: none;
      font-size: 1.3em;
    }

    div {
      display: flex;
      justify-content: space-around;
      align-items: center;
      width: 40%;
    }
  `;

  return (
    <StyledNav>
      <img src={logo} alt="img" />
      <div>
        <Router>
          <Link to="/">Home</Link>
          <Link to="/calc">Calculator</Link>
          <Link to="/quote">Quote</Link>
        </Router>
      </div>
    </StyledNav>
  );
};

export default Navbar;
