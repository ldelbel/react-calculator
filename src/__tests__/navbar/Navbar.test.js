import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../../components/Navbar';

it('renders correctly', () => {
  const navbar = renderer.create(<Navbar />).toJSON();
  expect(navbar).toMatchSnapshot();
});
