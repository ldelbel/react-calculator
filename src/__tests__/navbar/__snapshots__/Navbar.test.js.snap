// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="sc-bdfBwQ hjshVr"
>
  <img
    alt="img"
    src="math.png"
  />
  <div>
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/calc"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </div>
</nav>
`;
